src/shared/services/shortcut-action.service.ts

```diff
@@ -80,12 +80,14 @@ export class ShortcutActionService {
    * @param {number} direction - Movement direction. -1 for UP, +1 for DOWN
    */
   private move(path: Array<any>, direction: number): void {
+    this.domUtilService.blurFirstInputChildById(path.join('.'));
     let index = this.pathUtilService.getElementIndexInForwardOrReversePath(path, false);
     path[path.length - 2] = this.moveElement(index, direction, this.pathUtilService.getNearestOrRootArrayParentInPath(path, false));
     let pathString = path.join('.');
```

Comment:
can you do this before the https://github.com/inveniosoftware-contrib/ng2-json-editor/pull/165/files#diff-0da253bfd51f517c1804433a90ad5351R83 so that we don't `join` twice.